### ISSUE TITLE
24/192

### ISSUE THREAD

**Issue by hoosierdaddy716** [kabouzeid/phonograph-issue-tracker#56]
Please add ability to play 24/192 flac

**Comment by kabouzeid** [comment]
I don't think that this is possible, since I use the Android Media Player API to play files and have no control over what it can play and what not.
